Makefile

```diff
@@ -64,6 +64,25 @@ run-local:
 	./meshery; \
 	cd ..
 
+run-tests:
+	# -----Linting Check----- #
+	GO111MODULE=off GOPROXY=direct GOSUMDB=off go get github.com/mgechev/revive;
+	$(GOPATH)/bin/revive -config tools-config/revive-lint.toml -formatter friendly ./... \
+
+	# ------Error Check------ #
```

Comment:
Can this be removed ?

Reply:
Can you be more specific @kushthedude , not getting you here?
Wanna remove the linting check?

Reply:
The comments above the tests